portal/static/js/main.js

```diff
@@ -123,9 +123,9 @@ var fillContent = {
                 }
             };
             // Display clinics if any value is false (except localized) or if all are answered
-            if ((clinicalValue == "false" && clinicalItem != "pca_localized") || i == 3) {
-                $("#clinics").fadeIn();
-            }
+            //if ((clinicalValue == "false" && clinicalItem != "pca_localized") || i == 3) {
```

Comment:
Did you intend to leave these statements commented out?  If they aren't needed, can we remove them?